### PR TITLE
Send jacocoExecpath to sonar as a parameter

### PR DESCRIPTION
**Pull request by AshwiniChalla**
Testing done: Verified with/without code coverage tool, Cobertura, jacoco. If the options already have the property.

### CODE REVIEW

File: Tasks/Maven/mavenHelper.ps1 [hunk 1]
```diff
@@ -104,10 +105,15 @@ function RunSonarQubeAnalysis
...
 		}
-
+		
 		$sqArguments = $userOptions + " " + $sqArguments
 		Write-Verbose "Running Maven with goal sonar:sonar and options: $sqArguments"
-
+		
+		if($execFileJacoco)
+		{
+			$sqArguments = $sqArguments + " -Dsonar.jacoco.reportPath=" + $execFileJacoco
```
Comment (DavidStaheli) [inline Tasks/Maven/mavenHelper.ps1]: Have you tested this when $execFileJacoco contains spaces or special characters?  I wonder if the path needs to be wrapped in quotes/apostrophes.

Comment (AshwiniChalla) [inline Tasks/Maven/mavenHelper.ps1]: Makes sense. Will try adding quotes

File: Tasks/Maven/mavenHelper.ps1 [hunk 2]
```diff
@@ -104,10 +105,15 @@ function RunSonarQubeAnalysis
...
 		}
-
+		
 		$sqArguments = $userOptions + " " + $sqArguments
 		Write-Verbose "Running Maven with goal sonar:sonar and options: $sqArguments"
-
+		
+		if($execFileJacoco)
+		{
+			$sqArguments = $sqArguments + " -Dsonar.jacoco.reportPath=" + '"'+ $execFileJacoco +'"'
```
Comment (bgavrilMS) [inline Tasks/Maven/mavenHelper.ps1]: There is an EscapeArg function in this file that will do the escaping

**Comment by bgavrilMS** [comment]
LGTM

**Pull request merged**